src/App.js

```diff
@@ -57,4 +81,5 @@ function App() {
   }
 }
 
+
```

Comment:
@AkhileshManda 
Is there a reason for this extra whitespace?

Reply:
No reason @mrbazzan , removed it with my latest commit

Reply:
Nice!